### PR TITLE
Add dashboard search input component

### PR DESCRIPTION
**Pull request by viviandiec**
image

### CODE REVIEW

**Review by meganhng — APPROVED** [review]
Looks good!!

**Review by Txu045 — APPROVED** [review]

File: components/core/DashboardSearchInput.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        height="100%"
+        fontSize="16px"
+        background="white"
+        color="text.primary"
+        borderWidth="0 0 1px 0"
+        borderRadius="0"
+        _hover={{}}
+      />
+      <InputLeftElement height="100%" paddingRight="5px" paddingLeft="5px">
+        <SearchIcon></SearchIcon>
```
Comment (sttic) [inline components/core/DashboardSearchInput.tsx:23]: Nitpick: should be `<SearchIcon />` (self closing empty components)

File: components/core/DashboardSearchInput.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { Input, InputGroup, InputLeftElement, chakra } from "@chakra-ui/react";
+import SearchSvg from "../svg/search.svg";
+
+export const SearchIcon = chakra(SearchSvg);
+
+const DashboardSearchInput = () => (
+  <>
+    <InputGroup height="48px">
+      <Input
+        type="email"
```
Comment (sttic) [inline components/core/DashboardSearchInput.tsx:10]: The input isn't for emails?

**Review by sttic — APPROVED** [review]
I left some minor feedback :stuck_out_tongue: Looks good besides those nitpicks!

File: components/core/DashboardSearchInput.tsx (hunk 2 above)
Comment (viviandiec) [reply components/core/DashboardSearchInput.tsx:10]: oops

Comment (viviandiec) [reply components/core/DashboardSearchInput.tsx:10]: what type should i put it as?

Comment (sttic) [reply components/core/DashboardSearchInput.tsx:10]: I'd guess "search" type lol

**Pull request merged**